common/nymnoise/src/stream.rs

```diff
@@ -0,0 +1,202 @@
+// Copyright 2025 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::error::NoiseError;
+use bytes::BytesMut;
+use futures::{Sink, SinkExt, Stream, StreamExt};
+use pin_project::pin_project;
+use snow::{HandshakeState, TransportState};
+use std::cmp::min;
+use std::io;
+use std::pin::Pin;
+use std::task::Poll;
+use tokio::{
+    io::{AsyncRead, AsyncWrite, ReadBuf},
+    net::TcpStream,
+};
+use tokio_util::codec::{Framed, LengthDelimitedCodec};
+
+const MAXMSGLEN: usize = 65535;
```

Comment:
I've just noticed that both of those are defined inside `snow` instead. perhaps use those imports directly instead?

Reply:
Say what now ?!

Reply:
It's in a private module though, we can import it from there

Reply:
Since allocation behavior changed, this isn't needed anymore